AppCenterCrashes/AppCenterCrashesTests/MSCrashesUtilTests.m

```diff
@@ -29,32 +30,78 @@ - (void)tearDown {
 #pragma mark - Tests
 
 - (void)testCreateCrashesDir {
-  NSString *crashesDir = [[MSCrashesUtil crashesDir] path];
+
+  // If
   NSString *expectedDir;
-  XCTAssertNotNil(crashesDir);
+#if TARGET_OS_TV
+  expectedDir = @"/Library/Caches/com.microsoft.appcenter/crashes";
+#else
 #if TARGET_OS_OSX
-  expectedDir = @"/Library/Caches/com.test.app/com.microsoft.appcenter/crashes";
+  expectedDir = @"/Library/Application%20Support/(null)/com.microsoft.appcenter/crashes";
```

Comment:
I think it takes the info.plist from the test target, have you double checked if the app bundle ID was there? It can be missing.

Reply:
the app bundle is `nil` for test targets. It took me a while to figure that one out last night.